recipes/aws-libfabric/all/conandata.yml

```diff
@@ -0,0 +1,4 @@
+sources:
+  "1.9.1amzncdi1.0":
```

Comment:
1.11.2amzn1.1 is the most recent libfabric release: https://github.com/aws/libfabric/releases/tag/v1.11.2amzn1.1

Reply:
AWS CDI SDK, which is what I need this for, depends on the 1.9.x version. Should I add this new version as well?

Reply:
Isn't it compatible with 1.11.2?

Reply:
sorry, it's not any 1.9.x version but only those from the 1.9.x-cdi branch which are identified by having amzncdi in their name (which is currently only this release).

Reply:
Okay, don't forget to add this check in the AWS CDI SDK recipe then.